debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -334,6 +334,15 @@ public static SnapshotMode parse(String value, String defaultValue) {
             SNAPSHOT_MODE, CommonConnectorConfig.SOURCE_STRUCT_MAKER_VERSION,
             Heartbeat.HEARTBEAT_INTERVAL, Heartbeat.HEARTBEAT_TOPICS_PREFIX);
 
+    /** The comma-separated list of operations in MongoDB OpLog  that reader will skip reading them from OpLog */
+    public static final Field SKIP_OPLOG_OPEARATIONS = Field.create("skip.oplog.operations")
+            .withDisplayName("Operations")
+            .withType(Type.LIST)
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.LOW)
+            .withValidation(MongoDbConnectorConfig::validateSkipOperation)
+            .withDescription("The comma-separated list of operations in MongoDB OpLog ");
+
     protected static Field.Set EXPOSED_FIELDS = ALL_FIELDS;
```

Comment:
The field also need to go into `ALL_FIELDS` and `configDef()`.

Reply:
I did not find the `ALL_FIELDS` and either `configDef()` on the `CommonConnectorConfig`